### PR TITLE
feat(RHTAPWATCH-374): Run multiple Splunk queries

### PR DESCRIPTION
**Pull request by ifireball**
Add support in `querygen` for generating and printing multiple queries as well as support in `fetch-uj-records.sh` run running them.

In the process also add one additional query for querying PipelineRuns

### CODE REVIEW

File: cmd/querygen/main.go [hunk 1]
```diff
@@ -18,15 +20,35 @@ import (
...
+	if *machinePrint {
+		printFunc = queryprint.MachinePrintQueries
+	}
+	fmt.Println(printFunc([]queryprint.QueryDesc{
+		{
+			Title: "Application events",
+			Query: querygen.GenApplicationQuery(*index),
+		},
+		{
+			Title: "PipelineRun creations for build pipeeline",
```
Comment (Omeramsc) [inline cmd/querygen/main.go:50]: type: s/pipeeline/pipeline

File: querygen/querygen.go [hunk 2]
```diff
@@ -18,9 +18,10 @@ const (
 	// IncludeFieldsCmd is a Splunk query "fields" command for specifying which
 	// fields to include in the final query results. Since the structure of a
 	// Segment record is quite fixed, most queries should return the same
-	// fields.
+	// fields. Some queries may return a subset of these vields but not
```
Comment (Omeramsc) [inline querygen/querygen.go:21]: type: s/vields/fields

File: querygen/querygen.go [hunk 3]
```diff
@@ -119,5 +128,44 @@ func GenApplicationQuery(index string) string {
...
+		`log_type=audit ` +
+		`verb=create ` +
+		`"responseStatus.code" IN (200, 201) ` +
+		`"objectRef.apiGroup"="tekton.dev" ` +
+		`"objectRef.resource"="pipelineruns" ` +
+		`"responseObject.metadata.labels.pipelines.appstudio.openshift.io/type"=build` +
+		`"responseObject.metadata.resourceVersion"="*"` +
+		`|` + GenDedupEval(fields) +
+		`|` + GenTrackFields(track_fields, json_properties)
+}
```
Comment (yftacherzog) [inline querygen/querygen.go:171]: no newline at end of file

File: queryprint/queryprint.go [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+func MachinePrintQueries(queries []QueryDesc) string {
+	var builder strings.Builder
+	for i, query := range queries {
+		if i > 0 {
+			builder.WriteRune('\x00')
+		}
+		builder.WriteString(query.Query)
+	}
+	return builder.String()
+}
```
Comment (yftacherzog) [inline queryprint/queryprint.go:62]: newline at end of file

**Review by Omeramsc — APPROVED** [review]
LGTM

**Review by yftacherzog — APPROVED** [review]

**Pull request merged**